doc/exporter-guide.md

```diff
@@ -0,0 +1,49 @@
+# Exporter Developer Guide
+
+An exporter sends traces and metrics to any backend that is capable of consuming them. The exporter itself can change without requiring a change in your client code.
```

Comment:
What about the exporter changes? Does this mean you can change which exporter you are using or that the exporter itself might change? As a user, the wording "the exporter can change" makes me a little uncomfortable.
```suggestion
An exporter sends traces and metrics to any backend that is capable of consuming them. With OpenTelemetry, you can easily add and remove any exporter without requiring changes to your application code.
```